weave/trace_server/refs_internal.py

```diff
@@ -29,18 +31,15 @@ class InvalidInternalRef(ValueError):
     pass
 
 
-def quote_select(s: str, quote_chars: tuple[str, ...] = ("/", ":", "%")) -> str:
+def extra_value_quoter(s: str, quote_chars: tuple[str, ...] = ("%", "/")) -> str:
```

Comment:
I think this is correct - perhaps we just want to quote everything? `urllib.parse.quote(s, safe="")`? That might be the better approach